dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/settings/SettingsWrapper.java

```diff
@@ -51,6 +51,10 @@ public boolean isProvCollectionEnabled() {
         return settingService.isTrueForKey(SettingsServiceBean.Key.ProvCollectionEnabled);
     }
 
+    public Integer getMinutesUntilPasswordResetTokenExpires() {
```

Comment:
This method is already in the systemConfig why not just use that class? We should minimize the "entry points" so when we refactor it is easier to take care of 1 method instead of 2.